From 891e7a09218eacd0f9c8f77a867f58a03b465ea6 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Tue, 24 Nov 2020 12:05:20 +0100
Subject: [PATCH] Don't hardcode the path to the macOS SDK

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 729ad4a..8fe817e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,7 +71,7 @@ elseif (APPLE)
            message(FATAL_ERROR "Security Framework not found")
         endif ()
 
-        set(PLATFORM_LIBS ${SECURITY_LIB})
+        list(APPEND PLATFORM_LIBS "-framework Security")
     endif()
 else ()
     if (NOT BYO_CRYPTO)
-- 
2.29.2

